lib/core/form/form-base.module.ts

```diff
@@ -40,6 +40,7 @@
 import { A11yModule } from '@angular/cdk/a11y';
 import { FlexLayoutModule } from '@angular/flex-layout';
 import { ViewerModule } from '../viewer/viewer.module';
+import { InplaceMatFormFieldWrapperComponent } from './components/inplace-mat-form-field-wrapper/inplace-mat-form-field-wrapper.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Alfresco/alfresco-ng2-components/pullRequest?prid=8947659)
```suggestion
import { InplaceMatFormFieldWrapperComponent } from "./components/inplace-mat-form-field-wrapper/inplace-mat-form-field-wrapper.component";
```